src/svg-icons/arrow-drop-down.js

```diff
@@ -0,0 +1,15 @@
+/* eslint-disable */
```

Comment:
Okay, so do I just need to change `/* @eslint-disable */` to `// @flow weak`? I can't spot any other differences.